homeassistant/components/neptune_apex/light.py

```diff
@@ -0,0 +1,131 @@
+"""Support outlet status/control from Neptune Apex."""
+import logging
+
+import pynepsys
+
+from homeassistant.components.light import ATTR_EFFECT, SUPPORT_EFFECT, Light
+
+from . import NEPTUNE_APEX, NEPTUNE_APEX_COORDINATOR
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Parse through the Apex outlets and creates Outlet entities."""
+    apex = hass.data[NEPTUNE_APEX]
+    coordinator = hass.data[NEPTUNE_APEX_COORDINATOR]
+
+    async_add_entities(
+        Outlet(coordinator, apex, name)
+        for i, name in enumerate(coordinator.data.outlets)
+    )
+
+
+class Outlet(Light):
+    """Shoehorn the Apex outlet concept into a hass Light entity.
+
+    This feels gross, but the more you play with Apex, you realize that once you get past
```

Comment:
Please remove this part of the docstring. We want the docstring to describe the class not describe how it makes us feel.

Reply:
Fair. Should I include the long-winded description of why Outlet==Light from elsewhere in this PR instead? For now I'm just removing the paragraph.